editor.planx.uk/src/@planx/components/List/Public/index.tsx

```diff
@@ -65,6 +66,8 @@ const InputField: React.FC<Field> = (props) => {
       return <ChecklistFieldInput id={inputFieldId} {...props} />;
     case "date":
       return <DateFieldInput id={inputFieldId} {...props} />;
+    case "map":
+      return <MapFieldInput id={inputFieldId} {...props} />;
```

Comment:
It would be nice to also add some tests in `/List/Public/index.test.tsx`, similar to this - https://github.com/theopensystemslab/planx-new/pull/3519/commits/960a28531d208c1c9d10db266ec7cb6002a1531e

I'm not _quite_ sure how map interactions are going to work in react-testing-library. If it's not simple I'd be happy to pass this by for now, but if it's a case off adding a `MapField` to the test Zoo schema and just clicking a point within a div then it's worth it I think!

Reply:
Added this as a follow-up to the Trello checklist for a couple reasons:
- Gotta re-fresh my memory how it works to mix React & native web components using react-testing-library
- We aren't handling key test-able states here yet like "Inactive" card which makes it hard to simply add to existing test schema
- Based on FindProperty order dependency, we'll also have to expand mock test data a bit with some fake breadcrumbs/passport as if FindProp has already been seen ! 

All totally do-able, but don't want to hold this up longer at this stage while still feature flagged if that's okay

Reply:
Yep, on board with this 👌